test/cssModules.transform.js

```diff
@@ -0,0 +1,21 @@
+// Really naive, regex-based class name extractor
+
+module.exports = {
+  process: src => {
+    const classNames = [];
+
+    const classNameRegEx = /\.([_a-zA-Z0-9]+)(?=[\.:\[\s{])/gm;
```

Comment:
I don't think we have any classes that use `-` but it might be good to support it to avoid later confusion:
Just replacing `[_a-zA-Z0-9]` to `[_a-zA-Z0-9-]+` would work. (though even that is not technically correct, but only because it's too broad)

Reply:
Done.